akka-docs/src/main/paradox/typed/index-persistence-durable-state.md

```diff
@@ -0,0 +1,14 @@
+---
+project.description: Event Sourcing with Akka Persistence enables actors to persist your events for recovery on failure or when migrated within a cluster.
```

Comment:
```suggestion
project.description: Use of Event Sourcing with Akka Persistence enables actors to persist your events. This persistence is used for recovery on failure, or when migrating within a cluster.
```

Reply:
actually it was a mistake to describe this as event sourcing (copied it from the other doc and forgot to change). It's not. Changed the text to the following:

> Durable state with Akka Persistence enables actors to persist the latest version of the state. This persistence is used for recovery on failure, or when migrating within a cluster.